scripts/canvas-script.js

```diff
@@ -633,6 +646,88 @@ clearConnectionButton.addEventListener("click", () => {
     buttonMenager.clearConnection = !buttonMenager.clearConnection
 })
 
+saveForm.addEventListener("click", () => {
+    var newData = new Object
+    var form = document.forms[0].elements
+
+    newData.name = form.name.value
+    newData.surname = form.surname.value
+    newData.sex = form.gender.value
+    newData.nationality = form.nationality.value
+    newData.birth_place = form.birthPlace.value
+    newData.birth_date = form.birthDate.value
+    newData.death_date = form.deathDate.value
+    newData.death_cause = form.deathCause.value
+    Api.patchPersonById(selectedPersonId, newData)
```

Comment:
var newData= Object.fromEntries(new FormData(document.forms[0]));
To to samo jakbyś miał atrybuty name takie same jak w api